mobile/src/bundles/common/styles/global-styles.ts

```diff
@@ -0,0 +1,89 @@
+import { type FlexAlignType, type FlexStyle, StyleSheet } from 'react-native';
+
+import { AlignmentValues } from '~/bundles/common/enums/enums';
+
+const globalStyles = StyleSheet.create({
+    defaultMarginVertical: {
+        marginVertical: 20,
+    },
+    defaultMarginHorizontal: {
+        marginHorizontal: 20,
+    },
+    defaultMarginTop: {
+        marginTop: 20,
+    },
```

Comment:
Let's add some values more values to margins and paddings, e.x. m5 = margin: 5, m10 = margin: 10, and so on till 20-25 (I guess there is no need for a bigger value). Same with margin-top/bottom/vertical etc.